src/hnsw.h

```diff
@@ -354,7 +354,7 @@ typedef struct HnswVacuumState
 	/* Variables */
 	struct tidhash_hash *deleted;
 	BufferAccessStrategy bas;
-	HnswNeighborTuple ntup;
+	HnswNeighborTuple ntupscratch;
```

Comment:
It took me a few minutes in RepairGraphElement() to figure out whether `vacuumstate->ntup` was initialized somewhere in the caller, until I understood that it's just a scratch space to avoid repeated palloc+pfrees in RepairGraphElement. This renaming makes it a little more clear.